oeps/best-practices/oep-0066/Open_edX_Authorization_Systems_Table.rst

```diff
@@ -0,0 +1,182 @@
+Open edX Authorization Systems Tables
+######################################
+
+
+.. list-table:: Open edX Authorization Systems Explicit Roles Table
+   :widths: 10 18 18 18 18 18
+   :header-rows: 1
+
+   * - 
+     - edX Platform - Course Roles
+     - comment_client - Discussion Roles
+     - edX Platform - Django Admin Flow
+     - edX rbac
+     - content_libraries - v2 Library Roles
+   * - System Users
```

Comment:
[Sorry for the piecemeal high-level comments.]
Do you think it is important to compare each of the rows at a glance across the columns? I ask because this table is wide and contains a lot of information. An alternative format would be to repeat the row headers (same order) for each authorization system, and have a separate row (or title) for each authorization type. I mean something similar to the tables under these different doc types, starting with [OEPS](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#oeps).

Reply:
For accessibility, either approach is OK IMO. When navigating tables, as you move the virtual cursor from one data cell to another (up/down or left/right), the new row or column header will be along with the data in that cell. So the linear reading experience might be more similar than it appears.

The separate row headers mapping is a useful one for linear reading of the table on narrow viewports, but that isn't actually a WCAG requirement; under the WCAG 2.1 Reflow requirement, data tables get an exception.

Reply:
The ability to compare the systems to each other is an important option, but I think the diagram allows for that. 

Given that both options are acceptable from an accessibility point of view and that the diagram exists for comparing across systems, I will update the table to match the suggested styles. The suggested styles will help increase understanding of each system and the diagram can be used for comparisons across systems.